tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -0,0 +1,121 @@
+#
+# Topology for digital microphones array
+#
+
+include(`platform/intel/dmic.m4')
+
+# variable that need to be defined in upper m4
+# define(CHANNELS, `4')
+
+# define(DMIC_PCM_48k_ID, `99')
+# define(DMIC_PIPELINE_48k_ID, `4')
+# define(DMIC_DAI_LINK_48k_ID, `1')
+
+# define(DMIC_PCM_16k_ID, `7')
+# define(DMIC_PIPELINE_16k_ID, `9')
+# define(DMIC_PIPELINE_KWD_ID, `10')
+# define(DMIC_DAI_LINK_16k_ID, `2')
+
+# define(KWD_PIPE_SCH_DEADLINE_US, 20000)
+
```

Comment:
Can we remove all the static definitions from this file and move it to respective .m4?
That will give the developer flexibility to define the pipelines on the platform need

Reply:
this is example code. Please note the `#`